x/posts/legacy/v5/store.go

```diff
@@ -0,0 +1,121 @@
+package v5
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	v4 "github.com/desmos-labs/desmos/v4/x/posts/legacy/v4"
+	"github.com/desmos-labs/desmos/v4/x/posts/types"
+)
+
+// MigrateStore performs the migration from version 4 to version 5 of the store.
+// To do this, it iterates over all the post attachments, and converts them to
+// the new storing format (AttachmentContent instead of Attachment).
+// It also removes all the Polls that have been saved as a Poll_ProvidedAnswer's attachment.
+func MigrateStore(ctx sdk.Context, storeKey sdk.StoreKey, cdc codec.BinaryCodec) error {
+	store := ctx.KVStore(storeKey)
+
+	// Migrate the poll attachments
```

Comment:
We should migrate media attachments as well since it implements `AttachmentContent` interface which is migrated to v3 from v2.

Reply:
Added the migration for the media attachments, thanks!